### PR TITLE
customized DoubleMuPt20Eta-27_27_cfi.py

### PR DESCRIPTION
**Pull request by anpicci**

### CODE REVIEW

File: python/Guns/DoubleMuPt20Eta-27_27_cfi.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        PartID = cms.vint32(13),
+        MaxEta = cms.double(2.7),
+        MaxPhi = cms.double(3.14159265359),
+        MinEta = cms.double(-2.7),
+        MinPhi = cms.double(-3.14159265359) ## in radians
+
+    ),
+    Verbosity = cms.untracked.int32(0), ## set to 1 (or greater)  for printouts
+
+    psethack = cms.string('single mu pt 15'),
```
Comment (agrohsje) [inline python/Guns/DoubleMuPt20Eta-27_27_cfi.py:16]: This line is still wrong. Or do I misunderstand?

**Comment by agrohsje** [comment]
Can you comment on why you like to delete the other 2 files ?

File: python/Guns/DoubleMuPt20Eta-27_27_cfi.py (hunk 1 above)
Comment (anpicci) [reply python/Guns/DoubleMuPt20Eta-27_27_cfi.py]: Sorry for the late reply

I've corrected that two lines and I've committed the new corrected version

**Comment by anpicci** [comment]
Because my local cmssw repo fails to compile them, I've thought deleting them in localy would not implying deleting them on github, sorry

**Comment by agrohsje** [comment]
It still shows up as deleted. Can you put them back, please?

**Comment by agrohsje** [comment]
Committed your file:
https://github.com/cms-sw/genproductions/commit/e5fba0073eeb3c65af200cbb25476b4168af3232
Will close this PR.

**Pull request closed without merging**